preprocessors/action.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://image.a11y.mcgill.ca/preprocessors/action-recognition.schema.json",
+    "type": "object",
+    "title": "Human Action Recognition",
+    "description": "Preprocessor to detect human action in an image. The associated information can be found in preprocessor named 'ca.mcgill.a11y.image.preprocessor.actionrecognition'. ",
+    "properties":{
+        "actions":{
+            "type":"array",
+            "description": "Contains a list of actions detected by the preprocessor 'ca.mcgill.a11y.image.preprocessor.actionRecognition' or empty list if no actions are found",
```

Comment:
Be more specific about these actions since this is a list of objects. Also note that this schema is `ca.mcgill.a11y.image.preprocessor.actionRecognition`, the preprocessor is just one module that can produce information on actions in that format.

Reply:
Should I reword this? Is there a better way to refer to the preprocessor?